scrapers/Karissa-Diamond.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Karissa Diamond'
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - karissa-diamond.com/videoCollection
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@id="setTitle"]/text()
+      Studio:
+        Name:
+          fixed: "Karissa Diamond"
+      Date:
+        selector: //div[@id="setDate"]/text()
+        postProcess:
+          - parseDate: January 02, 2006
+      Details: //div[@class="well"]/text()
+      Tags:
+        Name: //div[@class="col-md-5 col-md-offset-1 col-md-push-6 hidden-xs"]//div[@class="col-xs-12 tagList"]/a/text()
```

Comment:
This would be shorter and more future proofed for this this class names change a little:

```
//div[@class[contains(.,'tagList')]]/a/text()
```

Reply:
Ahh thanks. Still relatively new to XPaths.